manifests/n/Natron/Natron/2.5.0/Natron.Natron.installer.yaml

```
@@ -0,0 +1,30 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Natron.Natron
+PackageVersion: 2.5.0
+Installers:
+- InstallerUrl: https://github.com/NatronGitHub/Natron/releases/download/v2.5.0/Natron-2.5.0-Windows-x86_64.zip
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: inno
+  NestedInstallerFiles:
+  - RelativeFilePath: Natron-2.5.0-Windows-x86_64\Setup.exe
+  InstallerSha256: CAED0B649FF2515B963F0B4FC009EB451A41D8BA3D6E5D4260253DD9E104C517
+  Scope: machine
+  ElevationRequirement: elevationRequired
+  InstallerSwitches:
+    Custom: /ALLUSERS
+- InstallerUrl: https://github.com/NatronGitHub/Natron/releases/download/v2.5.0/Natron-2.5.0-Windows-x86_64.zip
+  Architecture: x86

```

```suggestion
  Architecture: x64
```
Since `x86` validations are performed on a 32-bit OS, I'm guessing this is 64-bit.